game/gbtd_gbr0.ksy

```diff
@@ -0,0 +1,217 @@
+meta:
+  id: gbr0
+  endian: le
+  encoding: ascii
+  title: GBR file
+  license: MIT
+  application: GameBoy Tile Designer, GBTD
+  file-extension:
+    - gbr
+  tags:
+    - game
+
+doc: |
+  GBR0 files are used by GameBoy Tile Designer
+  to store tilemaps created with the program.
+doc-ref: http://www.devrs.com/gb/hmgd/gbtd.html
+
+seq:
+  - id: magic
+    contents: [0x47, 0x42, 0x4F, 0x30] #GBO0
+  - id: objects
+    type: gbr_object
+    repeat: eos
+
+types:
+  gbr_object:
+    seq:
+      - id: object_type
+        type: u2
+      - id: object_id
+        type: u2
+      - id: record_length
+        type: u4
+      - id: body
+        size: record_length
+        type:
+          switch-on: object_type
+          cases:
+            0x0001: object_producer
```

Comment:
The constants should be enums.

Reply:
Never used them before, tried doing it. Is it correct?

Reply:
LGTM